### ISSUE TITLE
how  many epoches is properly?

### ISSUE THREAD

**Issue by wzf19947** [AlexeyAB/darknet#8734]
I usually trained my model with 200~300 epoch, which both got not so good results, and from the loss chart I can't tell if it's overfit or unfit ,  is there any idea for measuring that? Thanks.

**Comment by stephanecharette** [comment]
Darknet doesn't use epochs.  I hope you don't mean max batches?  The *minimum* max batches needed for Darknet is 6000 for 1 class, so "200-300" would definitely not be enough.

Please see the FAQ:  https://www.ccoderun.ca/programming/darknet_faq/#tldr

**Comment by wzf19947** [comment]
> Darknet doesn't use epochs. I hope you don't mean max batches? The _minimum_ max batches needed for Darknet is 6000 for 1 class, so "200-300" would definitely not be enough.
> 
> Please see the FAQ: https://www.ccoderun.ca/programming/darknet_faq/#tldr

I know what , my  "epoch" means maxbatch*batchsize/total_training_pic , right?  I just want to know how many of that is proper without overfit or unfit,thanks.
